src/html/components/concept.html

```diff
@@ -0,0 +1,7 @@
+<div class="concept" id="concept">
```

Comment:
Another thing you could do with the markup is to use `<section>` elements for major parts of the page, like for each of these main page blocks. It's like a div, but adds a bit of semantic meaning to the structure.